### PR TITLE
feat(component): split section header

### PR DESCRIPTION
**Pull request by gepifanio**
- [x] Still, need to go through the zero height recommendations and make sure all the dot points have been addressed.

https://zeroheight.com/17lwnkf17/p/1716a8

### CODE REVIEW

**Comment by gepifanio** [comment]
Comment: Deploy Started

Building with commit cb1d9d6b9b1f595dba65ef45cf3a5f5b0291d18b

https://app.netlify.com/sites/pattern-lib-unimelb/deploys/5d5ceb6f3203ef000852861e

File: components/header/SplitSectionHeader.vue [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<template>
+  <div
+    :style="{ backgroundImage: `url(${img})` }"
+    class="split-section-header">
+    <div class="split-section-header__container">
+      <div class="split-section-header__inner">
+        <div class="split-section-header-content">
```
Comment (TimMansell) [inline components/header/SplitSectionHeader.vue]: use `split-section-header__content`

File: components/header/split-section-header.css [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    position: relative;
+
+    &::before {
+      position: absolute;
+      content: '';
+      background-color: rgb(var(--col-background-tertiary-navy), .92);
+      width: 100vw;
+      height: 115vw;
+      top: 0;
+      left: -25px;
```
Comment (TimMansell) [inline components/header/split-section-header.css]: use rem

File: components/header/split-section-header.css [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+      @media (--bp-desktop) {
+        transform: rotate(15deg);
+        transform-origin: top right;
+        top: -80px;
```
Comment (TimMansell) [inline components/header/split-section-header.css]: use rem

**Comment by TimMansell** [comment]
Ready for review @jonathan-doc

**Comment by TimMansell** [comment]
- [ ] Fix layout on IE11

Screen Shot 2019-08-26 at 11 09 38 am

**Comment by gepifanio** [comment]
@TimMansell thanks for picking it up, has been fixed the IE11 issue:

image

**Comment by jonathan-doc** [comment]
**Padding-right** for split-section-header__inner:

- [ ] **18vw** between 769-900w breakpoints
- [ ] **13vw** for widths greater than 900

Revert to Mobile layout for widths less than 769 (already behaves like this, so that's fine)

- [ ] Fix background showing on widths between 320 and 357.

**Comment by TimMansell** [comment]
@gepifanio

**Pull request merged**